js/galleryfileaction.js

```diff
@@ -210,6 +216,13 @@ $(document).ready(function () {
 	var url = window.galleryFileAction.buildGalleryUrl('config', '', {extramediatypes: 1});
 	$.getJSON(url).then(function (config) {
 		window.galleryFileAction.buildFeaturesList(config.features);
+
+		if (config.mediatypes.length === 0) {
```

Comment:
I think it is sufficient to detect it this way, because no thumbnails are generated. My pr only affects the file viewer, on the gallery page it still says "No pictures found".